### DISCUSSION TITLE
Welcome to locast2plex Discussions!

### DISCUSSION THREAD

**Discussion by tgorgdotcom** [tgorgdotcom/locast2plex#102] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tharris9d** [comment]
Thomas, Thanks so much for adding this. It will be a big help posting items for the community!

**Comment by chris00780** [comment]
Indianapolis was recently added to locast. Does this script need to be updated to work with new markets? Thanks!

**Comment by joerich** [comment]
Dude! I just gotta say WOW! 
I just LOVE this addon to plex!
setup was pretty easy for me and I love the ability to record my favorite TV shows!
YOU ARE AWESOME! Thanks!
